src/app/components/profile/profile.component.html

```diff
@@ -0,0 +1,31 @@
+<mat-card>
+  <mat-card-header>
+    <mat-card-title>Name</mat-card-title>
+  </mat-card-header>
+  <mat-card-content>
```

Comment:
please add all fields in one mat-card-content, just use the list.

Reply:
Done